### PR TITLE
Updated Locaiton Adding UI

### PR DESCRIPTION
**Pull request by jfangrad**
Update the UI of the location creation popup according to the new UI
screen shot 2018-09-10 at 9 22 27 pm

### CODE REVIEW

**Comment by jfangrad** [comment]
[Buttons](https://trello.com/c/W5WeQDlL/137-buttons)

**Comment by jfangrad** [comment]
[Header Colour](https://trello.com/c/c2J6sOUd/139-header-colour)

**Comment by smaclell** [comment]
@jfangrad should this also include: https://trello.com/c/gb3tGeO0

File: components/ResourceCounter.js [hunk 1]
```diff
@@ -56,7 +56,17 @@ class ResourceCounter extends Component {
...
+        <Stepper
+          initialValue={this.props.initialCount}
+          displayValue
+          maximumValue={999}
+          minimumValue={0}
+          onValueChange={this.onCountChanged}
+          borderColor={colours.blues.accent}
+          tintColor={colours.blues.accent}
+          textColor={colours.blues.accent}
+          borderWidth={2}
```
[the remark this answers is not in the document]
Comment (jfangrad) [reply components/ResourceCounter.js]: No cause its a dependency we import and it doesn't accept a style object prop 😕

**Comment by jfangrad** [comment]
Updated UI:
screen shot 2018-09-21 at 9 18 04 pm
screen shot 2018-09-21 at 9 18 20 pm

In Seperate PR we will have to find a way to make the counter look like the new UI

File: reducers/statuses.js [hunk 2]
```diff
@@ -8,24 +8,24 @@ const yellow = '#ffdd00';
...
   {
-    key: 'accepted', label: 'status/accepted', icon: 'book', iconFamily: 'font-awesome', pinColor: green, group: 'home', success: true,
+    key: 'accepted', label: 'status/accepted', icon: 'check-circle', iconFamily: 'feather', pinColor: green, group: 'home', success: true,
   },
   {
     key: 'need', label: 'status/need', icon: 'exclamation-circle', iconFamily: 'font-awesome', pinColor: blue, group: 'home', success: false,
   },
   {
-    key: 'have', label: 'status/have', icon: 'check-circle', iconFamily: 'font-awesome', pinColor: green, group: 'home', success: true,
+    key: 'have', label: 'status/have', icon: 'open-book', iconFamily: 'entypo', pinColor: green, group: 'home', success: true,
```
Comment (jfangrad) [inline reducers/statuses.js]: Couldn't find this one on feather unfortunately

File: styles/fonts.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 export default {
```
Comment (smaclell) [inline styles/fonts.js]: We should have them scale at a normal rate. That way they visually look proportional.

**Comment by smaclell** [comment]
Did the designs have a border radius at the top?

**Comment by jfangrad** [comment]
@smaclell yes it does

**Pull request merged**